<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_AdminsSectionWindow</class>
 <widget class="QDialog" name="admin_AdminsSectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Search_Button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Search_LineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>409</width>
      <height>429</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="Role_Search">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>91</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="EditMember_Button">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>620</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteMember_Button">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>620</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="role_filter_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Role Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Logout_Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>625</y>
     <width>170</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>320</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>329</width>
      <height>269</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Profile_Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>170</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admins_Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>170</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string>Admins</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Members_Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>170</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string>Members</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Books_Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>170</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string>Books</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddMember_Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>620</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteAccount_Button">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>620</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditAccount_Button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>620</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_account">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>399</width>
      <height>279</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ReturnOrReserveBook_Button">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>620</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <zorder>AddMember_Button</zorder>
  <zorder>DeleteMember_Button</zorder>
  <zorder>EditMember_Button</zorder>
  <zorder>scrollArea</zorder>
  <zorder>scrollArea_2</zorder>
  <zorder>Search_Button</zorder>
  <zorder>Search_LineEdit</zorder>
  <zorder>Role_Search</zorder>
  <zorder>role_filter_label</zorder>
  <zorder>Logout_Button</zorder>
  <zorder>Profile_Button</zorder>
  <zorder>Admins_Button</zorder>
  <zorder>Members_Button</zorder>
  <zorder>Books_Button</zorder>
  <zorder>DeleteAccount_Button</zorder>
  <zorder>EditAccount_Button</zorder>
  <zorder>scrollArea_account</zorder>
  <zorder>ReturnOrReserveBook_Button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
